pubsublite/admin/update_topic.go

```diff
@@ -36,7 +36,12 @@ func updateTopic(w io.Writer, projectID, region, zone, topicID string) error {
 	}
 	defer client.Close()
 
-	topicPath := fmt.Sprintf("projects/%s/locations/%s/topics/%s", projectID, zone, topicID)
+	var topicPath string
+	if regional {
+		topicPath = fmt.Sprintf("projects/%s/locations/%s/topics/%s", projectID, region, topicID)
+	} else {
+		topicPath = fmt.Sprintf("projects/%s/locations/%s/topics/%s", projectID, zone, topicID)
+	}
 	// For ranges of fields in TopicConfigToUpdate, see https://pkg.go.dev/cloud.google.com/go/pubsublite/#TopicConfigToUpdate
 	config := pubsublite.TopicConfigToUpdate{
```

Comment:
Can we also show ThroughputReservation as one of the field to be updated in this update topic example? Sorry I didn't think of this one in the first pass.

Reply:
Why is the `regional` bool missing from the Python/Java samples for update topic? I also noticed it is missing for get topic as well.

Reply:
Update topic Java/Python samples have been updated to use a bool for regional too. Thanks for pointing that out!